File: 0344-reverse-string/0344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int first = 0;
        int last = s.size() - 1;
        while(first<last){
            char temp = s[last];
            s[last] = s[first];
            s[first]= temp;
            first++;
            last--;
        }

    }
};

File: 0344-reverse-string/0344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        first = 0
        last = len(s) - 1
        while first <= last:
            temp = s[last]
            s[last] = s[first]
            s[first] = temp
            first += 1
            last -= 1
        
        return s
        